src/templates/actors/drone-sheet.html

```diff
@@ -0,0 +1,278 @@
+<form class="{{cssClass}} bg-gray-200 -m-2 p-2 space-y-2">
+  <div>
+    <div class="flex gap-2">
+      <div class="flex-shrink-0 h-16 hidden md:block md:w-16">
+        <img
+          class="shadow-md bg-gray-100 h-full w-full object-contain"
+          src="{{actor.img}}"
+          title="{{actor.name}}"
+          data-edit="img"
+        />
+      </div>
+      <div class="flex-grow grid grid-cols-1 lg:grid-cols-2 items-center gap-2">
+        <input
+          class="text-2xl px-1.5 py-3.5 border border-gray-800 bg-gray-400 bg-opacity-75 placeholder-blue-800 placeholder-opacity-75 rounded-md"
+          name="name"
+          type="text"
+          value="{{actor.name}}"
+          placeholder="{{localize 'swnr.sheet.name'}}"
+        />
+        <h2 class="flex items-center gap-2">
+          <select
+            class="model-select flex-grow w-full border border-gray-800 bg-gray-400 bg-opacity-75 placeholder-blue-800 placeholder-opacity-75 rounded-md"
+            name="data.model"
+          >
+            {{#select data.model}}
+            <option value="primitiveDrone">
+              {{localize 'swnr.drone.model.primitiveDrone.name'}}
+            </option>
+            <option value="voidHawk">
+              {{localize 'swnr.drone.model.voidHawk.name'}}
+            </option>
+            <option value="stalker">
+              {{localize 'swnr.drone.model.stalker.name'}}
+            </option>
+            <option value="cuttlefish">
+              {{localize 'swnr.drone.model.cuttlefish.name'}}
+            </option>
+            <option value="ghostwalker">
+              {{localize 'swnr.drone.model.ghostwalker.name'}}
+            </option>
+            <option value="sleeper">
+              {{localize 'swnr.drone.model.sleeper.name'}}
+            </option>
+            <option value="pax">
+              {{localize 'swnr.drone.model.pax.name'}}
+            </option>
+            <option value="alecto">
+              {{localize 'swnr.drone.model.alecto.name'}}
+            </option>
+            <option value="other">
+              {{localize 'swnr.drone.model.other.name'}}
+            </option>
+            {{/select}}
+          </select>
```

Comment:
Again, this is a code smell and locks out custom content. I do not want to treat 3rd party content as a second class citizen.

Reply:
That makes sense. Leaving code like this general will also help maintain developers' sanity.

I left an "Other" option in place to allow someone who doesn't want to or cannot modify the compendium to specify a custom model. Let me know what you think.